x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -583,6 +583,9 @@ export class FleetPlugin
           }
         );
 
+        // initialize (generate/encrypt/validate) Uninstall Tokens asynchronously
+        this.initializeUninstallTokens();
```

Comment:
The change looks good to me, my only concern is what happens if there is an unexpected error coming from this function, it would stop the Fleet plugin from starting. We should probably make sure to catch any possible errors.

Reply:
that makes sense, thanks for the suggestion!
added it here, and unified the error handling of the two functions for generation and validation: a1707523a6e8a805b774593c3aceeeab0cf2156c